CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.54.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.53.0...v1.54.0) (2024-01-22)
+
```

Comment:
Deduplicate the changelog - due to missing commits in `main` branch, 1.53.0 version changes got duplicated

Reply:
updated